great_international/serializers.py

```diff
@@ -2414,14 +2424,23 @@ class InvestmentOpportunityPageSerializer(BasePageSerializer, HeroSerializer):
         AVATAR_RENDITION_SPEC,
     )
 
-    # Relations
-    related_regions = RegionPageSummarySerializer(
-        many=True,
-        read_only=True,
-    )
     related_sectors = serializers.SerializerMethodField()
     sub_sectors = serializers.SerializerMethodField()
     related_opportunities = serializers.SerializerMethodField()
+    related_regions = serializers.SerializerMethodField()
+
+    def _get_regions(self, instance):
+        return set([item.value['region'] for item in instance.regions_with_locations])
```

Comment:
Because 'regions' is not a mandatory field in the structblock, it's possible someone will save bad data and make this blow up. Not an approval blocker, but worth a tweak - this might work:
```suggestion
        return set([item.value['region'] for item in instance.regions_with_locations if item.value['region']])
```

Reply:
I guess the other helpers that do this will also benefit from the fix, too